py-polars/tests/unit/operations/namespaces/test_strptime.py

```diff
@@ -151,6 +151,22 @@ def test_to_date_all_inferred_date_patterns(time_string: str, expected: date) ->
     assert result[0] == expected
 
 
+@pytest.mark.parametrize(
+    ("time_string", "expected"),
+    [
+        ("2024-12-04 09:08:00", datetime(2024, 12, 4, 9, 8, 0)),
+        ("2024-12-4 9:8:0", datetime(2024, 12, 4, 9, 8, 0)),
+        ("2024/12/04 9:8", datetime(2024, 12, 4, 9, 8, 0)),
+        ("4/12/2024 9:8", datetime(2024, 12, 4, 9, 8, 0)),
+    ],
+)
+def test_to_datetime_infer_missing_digit_in_time(
```

Comment:
could you include the issue number in the test title please?

Reply:
added, thanks for the review